tiledb/tests/test_fixes.py

```diff
@@ -209,9 +209,17 @@ def get_config_with_env(env, key):
             )
             return sp_output.decode("UTF-8").strip()
 
-        assert get_config_with_env({}, "vfs.s3.region") == "us-east-1"
         assert get_config_with_env({"AWS_DEFAULT_REGION": ""}, "vfs.s3.region") == ""
+        assert (
+            get_config_with_env({"AWS_DEFAULT_REGION": "us-east-1"}, "vfs.s3.region")
+            == "us-east-1"
+        )
+
         assert get_config_with_env({"AWS_REGION": ""}, "vfs.s3.region") == ""
+        assert (
+            get_config_with_env({"AWS_REGION": "us-east-1"}, "vfs.s3.region")
+            == "us-east-1"
+        )
```

Comment:
As an alternative to removing the test, can we condition the new assertions on `libtiledb.version()` being >= 2.27?

Reply:
You are right Teo. Thanks for the suggestion!